components/nexus-repository/src/main/java/org/sonatype/nexus/repository/view/Content.java

```diff
@@ -18,33 +18,45 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
+import org.sonatype.nexus.common.collect.AttributesMap;
 import org.sonatype.nexus.repository.view.Payload;
 
+import com.google.common.collect.Maps;
 import org.joda.time.DateTime;
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
 /**
- * Content, that is more than plain {@link Payload}
+ * Content, that is wrapped {@link Payload} with {@link Attributes}.
  *
  * @since 3.0
  */
 public class Content
     implements Payload
 {
-  private final Payload payload;
+  public static final String CONTENT_LAST_MODIFIED = "lastModified";
+
+  public static final String CONTENT_ETAG = "etag";
```

Comment:
Simply put, I could not find more proper place for them. 

They are attribute keys of a content, content itself does not use these, true. But if you look _where_ they are used (proxy facet support, maven proxy facet, maven facet, maven group facet...) the use is spread over multiple facets not quite tied logically to each other, to call out one of them as "the proper" place for these (and these are not maven specific at all). For now am collecting them here, and we can see where to move it.